### PR TITLE
add normalization for aishell2 recipe

### PR DESCRIPTION
**Pull request by shanguanma**
This pull will finish the #760  and add normalization for aishell2 dataset.

### CODE REVIEW

File: lhotse/recipes/aishell2.py [hunk 1]
```diff
@@ -15,6 +15,73 @@
...
+
+    ---->>
+    IC0001W0061 听流年
+    IC0001W0062 听BEAT IT
+    IC0001W0063 听独角戏
+    IC0001W0064 听心雨
+    IC0001W0065 听YESTERDAY ONCE MORE
+    IC0001W0066 听广岛之恋
+    IC0001W0067 听一生有你
+    IC0010W0228 Here's
```
Comment (pzelasko) [inline lhotse/recipes/aishell2.py:60]: I think you may have also wanted to make utterances like these uppercase?

Comment (pzelasko) [reply lhotse/recipes/aishell2.py:60]: e.g. "beat it" was changed into "BEAT IT", but "Here's" is still "Here's"

**Comment by pzelasko** [comment]
@csukuangfj can you review pls? after we merge this, I'll release lhotse 1.6

File: lhotse/recipes/aishell2.py (hunk 1 above)
Comment (shanguanma) [reply lhotse/recipes/aishell2.py:60]: Yes, you are right, I fixed the example.

File: lhotse/recipes/aishell2.py [hunk 2]
```diff
@@ -15,6 +15,73 @@
...
+    IC0975W0451 明年二月底小成
+    ID0114W0368 我感觉就是在不断拉抽屉
+    ID0115W0198 我公司员工不存在持有和泰创投股份的情况
+
+    """
+    new_line = []
+    line = list(line)
+    for i, char in enumerate(line):
+        if char == "'" and "\u4e00" <= line[i - 1] <= "\u9fff":
+            char = char.replace("'", "")
```
Comment (csukuangfj) [inline lhotse/recipes/aishell2.py:78]:
```suggestion
            char = ''
```

File: lhotse/recipes/aishell2.py [hunk 3]
```diff
@@ -15,6 +15,73 @@
 from lhotse.utils import Pathlike
 
 
+def text_normalize(line: str):
```
Comment (csukuangfj) [inline lhotse/recipes/aishell2.py:18]:
```suggestion
def text_normalize(line: str) -> str:
```

**Comment by csukuangfj** [comment]
Looks good to me. Just left some minor comments.

File: lhotse/recipes/aishell2.py (hunk 3 above)
Comment (shanguanma) [reply lhotse/recipes/aishell2.py:18]: Ok, I do it.

File: lhotse/recipes/aishell2.py (hunk 2 above)
Comment (shanguanma) [reply lhotse/recipes/aishell2.py:78]: I don't know why you suggest changing it

Comment (pzelasko) [reply lhotse/recipes/aishell2.py:78]: it looks like it's simpler and equivalent since `char == "'"` anyway

**Comment by pzelasko** [comment]
LGTM

**Pull request merged**